internal/api/middleware.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+	"os"
+
+	"github.com/nikaydo/final/internal/jwt"
+)
+
+var (
+	ErrConfiguration = errors.New("server configuration error")
+	ErrAuthError     = errors.New("authentication required")
+	ErrValidJwt      = errors.New("error validation jwt token")
+)
+
+func auth(next http.HandlerFunc) http.HandlerFunc {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		secretKey := os.Getenv("TODO_PASSWORD")
+		if len(secretKey) == 0 {
+			writeJSONResponse(w, MsgError{Error: ErrConfiguration.Error()}, http.StatusInternalServerError)
+			return
+		}
+		TODO_SECRET_JWT := os.Getenv("TODO_SECRET_JWT")
```

Comment:
Это обычная переменная
```suggestion
		todoSecretJwt := os.Getenv("TODO_SECRET_JWT")
```